backend/de.metas.business/src/main/java/de/metas/costrevaluation/CostRevaluationRepository.java

```diff
@@ -23,4 +38,89 @@ public List<I_M_CostRevaluationLine> retrieveLinesByCostRevaluationId(@NonNull f
 				.create()
 				.list();
 	}
+
+	public I_M_CostRevaluation getById(@NonNull final CostRevaluationId costRevaluationId)
+	{
+		return InterfaceWrapperHelper.load(costRevaluationId, I_M_CostRevaluation.class);
+	}
+
+	@NonNull
+	public CostRevaluation retrieveById(@NonNull final CostRevaluationId costRevaluationId)
+	{
+		final I_M_CostRevaluation costRevaluationPO = getById(costRevaluationId);
```

Comment:
u can simply use `record` instead of `costRevaluationPO`.
(just a personal taste)